### PR TITLE
DOC: run_calibration API doc

### LINKED ISSUE #446: DOC: API docs?
It seems that we don't have API docs?
If that isn't the case can someone point me to the API docs?
We need API docs so users can look at all the args/kwargs for some things (eg `run_calibration`)

### LINKED ISSUE #449: DOC: add pyFAI calib user docs as link

### PR DESCRIPTION
**Pull request by chiahaoliu**
This PR

1. Includes API doc for ``run_calibration`` and ``xrun``
2. Includes ``pyFAI`` calibration gui doc to our calibration manual
3. Polish other docs, turn important commands into code block for better visibility (e.g. proxy setup)

closes #449 
closes #446

### CODE REVIEW

File: xpdacq/xpdacq.py [hunk 1]
```diff
@@ -422,6 +430,63 @@ def beamtime(self, bt_obj):
...
+            Temporary subscriptions (a.k.a. callbacks) to be used on
+            this run. Default to None. For convenience, any of the
+            following are accepted:
+
+            * a callable, which will be subscribed to 'all'
+            * a list of callables, which again will be subscribed to 'all'
+            * a dictionary, mapping specific subscriptions to callables or
+              lists of callables; valid keys are {'all', 'start', 'stop',
+              'event', 'descriptor'}
+
```
Comment (CJ-Wright) [inline xpdacq/xpdacq.py]: extra space?

File: xpdacq/xpdacq.py [hunk 2]
```diff
@@ -422,6 +430,63 @@ def beamtime(self, bt_obj):
...
+        verify_write: bool, optional
+            Double check if the data have been written into database.
+            In general data is written in a lossless fashion at NSLS-II
```
Comment (CJ-Wright) [inline xpdacq/xpdacq.py]: at the

**Comment by chiahaoliu** [comment]
yup, forgot to add Simon as reviewer.

File: xpdacq/xpdacq.py (hunk 1 above)
Comment (chiahaoliu) [reply xpdacq/xpdacq.py]: This extra space is needed, for the list above
 (most of the doc is copied from ``bluesky`` doc)

File: docs/quickstart.rst [hunk 3]
```diff
@@ -55,7 +56,8 @@ Carry out the steps in this order to ensure a successful experiment.
   5. Run your experiment by running ``xrun(<sample>, <scanplan>)`` and your data will be saved automatically.
 
 
-These and many more things are explained below and elsewhere in the documentation.
+These and many more things are explained below and elsewhere in the documentation.`XPD-Users Google group
```
[the remark this answers is not in the document]
Comment (chiahaoliu) [reply docs/quickstart.rst]: Thanks. This line is included by accident.

**Pull request merged**